command/include.go

```diff
@@ -0,0 +1,44 @@
+package command
+
+import (
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/peak/s5cmd/v2/strutil"
+)
+
+// createIncludesFromWildcard creates regex strings from wildcard.
+func createIncludesFromWildcard(inputIncludes []string) ([]*regexp.Regexp, error) {
```

Comment:
This function is identical to the function in `command/exclude.go`. Could you please try to re-use the same function for both include and exclude? Also need to rename `exclude.go` if we can re-use it (since it will be used on both exclude and include, must use common file name smth like `wildcard.go`).